README.md

```diff
@@ -0,0 +1,32 @@
+# helloworld
+This project aims to create the most
+- robust
+- scalable
+- future-proof
+- secure
+- innovative
+- AI-infused
+- blockchain-y
+- aesthetically pleasing
+- concurrent
+- best-practiced
+- test-driven
+- deterministic
+- cool
+- non-deterministic
+- B2B SAAS
+- free
+- open-source
+- containerized
+- embedded
+
+... hello world script ever.
+
+Make a PR and join the future.
+
+Rules:
+- Each run must only print "Hello, world!"
+- The total runtime must be <=1 second.
```

Comment:
I don't agree with this.

1. Any non trivial python script takes more than a second.
2. Doesn't leave much room if we want to make this as insane as the charter lists.

Reply:
I put that as a very low baseline, but I feel like there should be some upper bound. Otherwise people could go crazy with it and the script would never run. Limitations might breed creativity

Reply:
Let's say 20s. I imagine some really insane stuff. 1s seems extremely conservative.

Reply:
Done